### PR TITLE
RM-55-remove-upper-bound-error-on-ratcheting-tests

### PR DESCRIPTION
**Pull request by ryantimjohn**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -66,9 +67,8 @@ def run(self) -> None:
                 f"{self.type_of_coverage} coverage used to be {high_water_mark}; "
                 f"down to {new_coverage}%.  Fix by viewing '{self.coverage_url}'")
         elif new_coverage > high_water_mark:
-            with open(self.coverage_file, 'w') as f:
-                f.write(str(new_coverage))
-            print(f"Just ratcheted coverage up to {new_coverage}%")
+            warnings.warn(f"WARNING: Your current coverage is higher than current high watermark.\n"
+            f"Please consider increasing high watermark to in {self.coverage_file} to {new_coverage}.")
```
Comment (ryantimjohn) [inline setup.py:71]: E128 continuation line under-indented for visual indent

Comment (ryantimjohn) [inline setup.py:71]: E501 line too long (103 > 100 characters)

**Comment by ryantimjohn** [comment]
@Brunope I tested these changes both by artificially lowering a high water mark and by testing against this branch:
https://github.com/bluelabsio/records-mover/pull/207

Both worked! Yay!

**Review by Brunope — APPROVED** [review]
Seems good. Realistically are we ever gonna see these warnings if it just runs on the builds in automation? Maybe as a once-per-release kind of thing we can check and manually update the coverage?

**Comment by ryantimjohn** [comment]
> Seems good. Realistically are we ever gonna see these warnings if it just runs on the builds in automation? Maybe as a once-per-release kind of thing we can check and manually update the coverage?

I think we're never going to see these warnings, you're right.

I also agree it's a good idea to update things at least once per release!

**Pull request merged**